### ISSUE TITLE
NormCap doesn't work with notification=false in Ubuntu

### ISSUE THREAD

**Issue by sojusnik** [dynobo/normcap#142]
When `notification=false` in `~/.config/normcap/settings.conf`, then the OCR doesn't work.

Using Ubuntu 21.04 and installed NormCap with `sudo apt install tesseract-ocr tesseract-ocr-eng tesseract-ocr-deu tesseract-ocr-rus libtesseract-dev libleptonica-dev python3-dev` and `pip install normcap`.

This should help:
```
$ normcap -V
19:48:55 - INFO    - normcap.app            - L:41  - Start NormCap v0.2.6
19:48:55 - DEBUG   - normcap.app            - L:42  - CLI command: /home/sojusnik/.local/bin/normcap -V
19:48:55 - DEBUG   - normcap.app            - L:43  - QT LibraryPaths: ['/home/sojusnik/.local/lib/python3.9/site-packages/PySide2/Qt/plugins']
QSslSocket:  OpenSSL 1.1.1g  21 Apr 2020 True
19:48:55 - INFO    - normcap.utils          - L:152 - Copy 0 traineddata files to config directory
19:48:55 - DEBUG   - normcap.app            - L:65  - System info:
{  'config_directory': PosixPath('/home/sojusnik/.config/normcap'),
   'desktop_environment': <DesktopEnvironment.GNOME: 1>,
   'display_manager': <DisplayManager.X11: 2>,
   'is_briefcase_package': False,
   'normcap_version': '0.2.6',
   'platform': 'linux',
   'pyside2_version': '5.15.2',
   'qt_version': '5.15.2',
   'screens': {  0: ScreenInfo(is_primary=True, device_pixel_ratio=1.0, geometry=Rect(left=0, top=0, right=1920, bottom=1080), index=0)},
   'tessdata_path': '/usr/share/tesseract-ocr/4.00/tessdata/',
   'tesseract_languages': ['deu', 'eng', 'osd', 'rus'],
   'tesseract_version': '4.1.1'}
19:48:55 - DEBUG   - normcap.base_window    - L:49  - Create window for screen 0
19:48:55 - DEBUG   - normcap.base_window    - L:248 - Set window for screen 0 to fullscreen
19:48:55 - DEBUG   - normcap.system_tray    - L:21  - Set up tray icon
19:48:56 - DEBUG   - normcap.update_check   - L:67  - Search for new version on https://pypi.org/pypi/normcap/json
19:48:56 - DEBUG   - normcap.downloader     - L:32  - Download https://pypi.org/pypi/normcap/json
19:48:56 - DEBUG   - normcap.update_check   - L:38  - Newest version: 0.2.6 (installed: 0.2.6)
19:48:59 - DEBUG   - normcap.main_window    - L:179 - Hide 1 window(s)
19:48:59 - INFO    - normcap.main_window    - L:272 - Take screenshot of position (620, 411, 1207, 543)
19:48:59 - DEBUG   - normcap.screengrab     - L:32  - Grab screen: ScreenInfo(is_primary=True, device_pixel_ratio=1.0, geometry=Rect(left=0, top=0, right=1920, bottom=1080), index=0)
19:48:59 - DEBUG   - normcap.screengrab     - L:73  - Use capture method: QT by screen
19:48:59 - DEBUG   - normcap.utils          - L:32  - Store debug image in: /tmp/normcap/2021-09-12_19-48-59_215270_raw_qt.png
19:48:59 - DEBUG   - normcap.screengrab     - L:106 - Screen scale factor: 1.0
19:48:59 - DEBUG   - normcap.screengrab     - L:107 - Image devicePixelRatio: 1.0
19:48:59 - DEBUG   - normcap.screengrab     - L:110 - Image crop box rect: (620, 411, 587, 132)
19:48:59 - DEBUG   - normcap.utils          - L:32  - Store debug image in: /tmp/normcap/2021-09-12_19-48-59_379823_cropped.png
19:48:59 - DEBUG   - normcap.main_window    - L:281 - Prepare image for OCR
19:48:59 - INFO    - normcap.enhance        - L:17  - Apply enhancements to image
19:48:59 - DEBUG   - normcap.enhance        - L:92  - Resize screenshot by factor 3.2
19:48:59 - DEBUG   - normcap.utils          - L:32  - Store debug image in: /tmp/normcap/2021-09-12_19-48-59_385251_enlarged.png
19:48:59 - DEBUG   - normcap.utils          - L:32  - Store debug image in: /tmp/normcap/2021-09-12_19-48-59_437822_padded.png
19:48:59 - DEBUG   - normcap.main_window    - L:290 - Perform OCR
19:48:59 - DEBUG   - normcap.ocr            - L:62  - Init tesseract with args: {'path': '/usr/share/tesseract-ocr/4.00/tessdata/', 'lang': 'eng', 'oem': 1, 'psm': 1}
19:49:00 - INFO    - normcap.ocr            - L:85  - PSM Mode: 1, OSM Mode: 1, Mean Conf: 70.65
19:49:00 - INFO    - normcap.main_window    - L:295 - Raw text from OCR:
wee voewws ed ee —s Me ere Lp ee ew ee /plugins' ] 5lSocket: OpenSSL 1.1.1g 21 Apr 2020 True :48:55 - INFO - normeap.utils - L:152 - Copy 0 traineddata files to conf :48:55 - DEBUG - normcap.app - L:65 - System info: 'config_directory': PosixPath('/home/sojusnik/.config/normcap'), 'desktop_environment': <DesktopEnvironment.GNOME: 1>, 'display_manager': <DisplayManager.X11: 2>, 'is_briefcase_package': False,
19:49:00 - DEBUG   - normcap.main_window    - L:296 - Result from OCR:
best_magic: 
image: <PySide2.QtGui.QImage(QSize(2038, 582),format=QImage::Format_RGB32,depth=32,devicePixelRatio=1,bytesPerLine=8152,sizeInBytes=4744464) at 0x7f05b7fbe500>
image_area: 1186116
image_size: (2038, 582)
lines: wee voewws ed ee —s Me ere Lp ee ew ee /plugins' ] 5lSocket: OpenSSL 1.1.1g 21 Apr 2020 True :48:55 - INFO - normeap.utils - L:152 - Copy 0 traineddata files to conf
:48:55 - DEBUG - normcap.app - L:65 - System info:
'config_directory': PosixPath('/home/sojusnik/.config/normcap'),
'desktop_environment': <DesktopEnvironment.GNOME: 1>,
'display_manager': <DisplayManager.X11: 2>,
'is_briefcase_package': False,
mean_conf: 71.96296296296296
mode: 1
num_blocks: 3
num_lines: 4
num_pars: 3
psm_opt: 1
rect: Rect(left=620, top=411, right=1207, bottom=543)
scale_factor: 1.0
scores: {}
screen: ScreenInfo(is_primary=True, device_pixel_ratio=1.0, geometry=Rect(left=0, top=0, right=1920, bottom=1080), index=0)
text: wee voewws ed ee —s Me ere Lp ee ew ee /plugins' ] 5lSocket: OpenSSL 1.1.1g 21 Apr 2020 True :48:55 - INFO - normeap.utils - L:152 - Copy 0 traineddata files to conf :48:55 - DEBUG - normcap.app - L:65 - System info: 'config_directory': PosixPath('/home/sojusnik/.config/normcap'), 'desktop_environment': <DesktopEnvironment.GNOME: 1>, 'display_manager': <DisplayManager.X11: 2>, 'is_briefcase_package': False,
transformed: 
words: 
level:5  | page_num:1  | block_num:1  | par_num:1  | line_num:1  | word_num:1  | left:108  | top:80   | width:100  | height:7    | conf:0  | text:wee
level:5  | page_num:1  | block_num:1  | par_num:1  | line_num:1  | word_num:2  | left:275  | top:68   | width:113  | height:28   | conf:7  | text:voewws
level:5  | page_num:1  | block_num:1  | par_num:1  | line_num:1  | word_num:3  | left:576  | top:68   | width:62   | height:29   | conf:2  | text:ed
level:5  | page_num:1  | block_num:1  | par_num:1  | line_num:1  | word_num:4  | left:639  | top:80   | width:59   | height:13   | conf:19 | text:ee
level:5  | page_num:1  | block_num:1  | par_num:1  | line_num:1  | word_num:5  | left:1062 | top:80   | width:36   | height:7    | conf:22 | text:—s
level:5  | page_num:1  | block_num:1  | par_num:1  | line_num:1  | word_num:6  | left:1238 | top:80   | width:234  | height:16   | conf:40 | text:Me
level:5  | page_num:1  | block_num:1  | par_num:1  | line_num:1  | word_num:7  | left:1486 | top:80   | width:105  | height:7    | conf:44 | text:ere
level:5  | page_num:1  | block_num:1  | par_num:1  | line_num:1  | word_num:8  | left:1624 | top:80   | width:53   | height:13   | conf:45 | text:Lp
level:5  | page_num:1  | block_num:1  | par_num:1  | line_num:1  | word_num:9  | left:1688 | top:80   | width:101  | height:13   | conf:41 | text:ee
level:5  | page_num:1  | block_num:1  | par_num:1  | line_num:1  | word_num:10 | left:1800 | top:80   | width:43   | height:7    | conf:52 | text:ew
level:5  | page_num:1  | block_num:1  | par_num:1  | line_num:1  | word_num:11 | left:1860 | top:80   | width:49   | height:13   | conf:43 | text:ee
level:5  | page_num:1  | block_num:2  | par_num:1  | line_num:1  | word_num:1  | left:80   | top:105  | width:214  | height:40   | conf:83 | text:/plugins'
level:5  | page_num:1  | block_num:2  | par_num:1  | line_num:1  | word_num:2  | left:264  | top:101  | width:30   | height:58   | conf:54 | text:]
level:5  | page_num:1  | block_num:2  | par_num:2  | line_num:1  | word_num:1  | left:80   | top:155  | width:189  | height:34   | conf:48 | text:5lSocket:
level:5  | page_num:1  | block_num:2  | par_num:2  | line_num:1  | word_num:2  | left:319  | top:159  | width:158  | height:37   | conf:92 | text:OpenSSL
level:5  | page_num:1  | block_num:2  | par_num:2  | line_num:1  | word_num:3  | left:502  | top:159  | width:132  | height:37   | conf:88 | text:1.1.1g
level:5  | page_num:1  | block_num:2  | par_num:2  | line_num:1  | word_num:4  | left:680  | top:159  | width:43   | height:30   | conf:95 | text:21
level:5  | page_num:1  | block_num:2  | par_num:2  | line_num:1  | word_num:5  | left:745  | top:159  | width:67   | height:37   | conf:95 | text:Apr
level:5  | page_num:1  | block_num:2  | par_num:2  | line_num:1  | word_num:6  | left:837  | top:159  | width:88   | height:30   | conf:96 | text:2020
level:5  | page_num:1  | block_num:2  | par_num:2  | line_num:1  | word_num:7  | left:947  | top:159  | width:90   | height:30   | conf:96 | text:True
level:5  | page_num:1  | block_num:2  | par_num:3  | line_num:1  | word_num:1  | left:80   | top:210  | width:128  | height:30   | conf:67 | text::48:55
level:5  | page_num:1  | block_num:2  | par_num:3  | line_num:1  | word_num:2  | left:235  | top:226  | width:15   | height:5    | conf:91 | text:-
level:5  | page_num:1  | block_num:2  | par_num:3  | line_num:1  | word_num:3  | left:278  | top:210  | width:88   | height:31   | conf:96 | text:INFO
level:5  | page_num:1  | block_num:2  | par_num:3  | line_num:1  | word_num:4  | left:459  | top:226  | width:14   | height:5    | conf:92 | text:-
level:5  | page_num:1  | block_num:2  | par_num:3  | line_num:1  | word_num:5  | left:501  | top:207  | width:289  | height:40   | conf:62 | text:normeap.utils
level:5  | page_num:1  | block_num:2  | par_num:3  | line_num:1  | word_num:6  | left:1019 | top:226  | width:14   | height:5    | conf:92 | text:-
level:5  | page_num:1  | block_num:2  | par_num:3  | line_num:1  | word_num:7  | left:1062 | top:210  | width:109  | height:31   | conf:89 | text:L:152
level:5  | page_num:1  | block_num:2  | par_num:3  | line_num:1  | word_num:8  | left:1198 | top:226  | width:15   | height:5    | conf:91 | text:-
level:5  | page_num:1  | block_num:2  | par_num:3  | line_num:1  | word_num:9  | left:1238 | top:210  | width:90   | height:37   | conf:96 | text:Copy
level:5  | page_num:1  | block_num:2  | par_num:3  | line_num:1  | word_num:10 | left:1350 | top:210  | width:23   | height:30   | conf:85 | text:0
level:5  | page_num:1  | block_num:2  | par_num:3  | line_num:1  | word_num:11 | left:1396 | top:207  | width:245  | height:34   | conf:91 | text:traineddata
level:5  | page_num:1  | block_num:2  | par_num:3  | line_num:1  | word_num:12 | left:1665 | top:207  | width:111  | height:34   | conf:96 | text:files
level:5  | page_num:1  | block_num:2  | par_num:3  | line_num:1  | word_num:13 | left:1799 | top:211  | width:45   | height:30   | conf:96 | text:to
level:5  | page_num:1  | block_num:2  | par_num:3  | line_num:1  | word_num:14 | left:1865 | top:207  | width:93   | height:34   | conf:95 | text:conf
level:5  | page_num:1  | block_num:2  | par_num:3  | line_num:2  | word_num:1  | left:80   | top:262  | width:128  | height:30   | conf:67 | text::48:55
level:5  | page_num:1  | block_num:2  | par_num:3  | line_num:2  | word_num:2  | left:235  | top:278  | width:14   | height:4    | conf:92 | text:-
level:5  | page_num:1  | block_num:2  | par_num:3  | line_num:2  | word_num:3  | left:275  | top:262  | width:113  | height:30   | conf:71 | text:DEBUG
level:5  | page_num:1  | block_num:2  | par_num:3  | line_num:2  | word_num:4  | left:459  | top:278  | width:14   | height:4    | conf:87 | text:-
level:5  | page_num:1  | block_num:2  | par_num:3  | line_num:2  | word_num:5  | left:501  | top:268  | width:245  | height:30   | conf:91 | text:normcap.app
level:5  | page_num:1  | block_num:2  | par_num:3  | line_num:2  | word_num:6  | left:1019 | top:278  | width:14   | height:4    | conf:91 | text:-
level:5  | page_num:1  | block_num:2  | par_num:3  | line_num:2  | word_num:7  | left:1062 | top:262  | width:87   | height:30   | conf:43 | text:L:65
level:5  | page_num:1  | block_num:2  | par_num:3  | line_num:2  | word_num:8  | left:1198 | top:278  | width:14   | height:4    | conf:84 | text:-
level:5  | page_num:1  | block_num:2  | par_num:3  | line_num:2  | word_num:9  | left:1240 | top:262  | width:134  | height:36   | conf:96 | text:System
level:5  | page_num:1  | block_num:2  | par_num:3  | line_num:2  | word_num:10 | left:1395 | top:258  | width:106  | height:34   | conf:96 | text:info:
level:5  | page_num:1  | block_num:3  | par_num:1  | line_num:1  | word_num:1  | left:104  | top:309  | width:412  | height:40   | conf:52 | text:'config_directory':
level:5  | page_num:1  | block_num:3  | par_num:1  | line_num:1  | word_num:2  | left:546  | top:309  | width:980  | height:41   | conf:82 | text:PosixPath('/home/sojusnik/.config/normcap'),
level:5  | page_num:1  | block_num:3  | par_num:1  | line_num:2  | word_num:1  | left:104  | top:360  | width:479  | height:41   | conf:51 | text:'desktop_environment':
level:5  | page_num:1  | block_num:3  | par_num:1  | line_num:2  | word_num:2  | left:611  | top:360  | width:577  | height:40   | conf:91 | text:<DesktopEnvironment.GNOME:
level:5  | page_num:1  | block_num:3  | par_num:1  | line_num:2  | word_num:3  | left:1219 | top:364  | width:60   | height:36   | conf:91 | text:1>,
level:5  | page_num:1  | block_num:3  | par_num:1  | line_num:3  | word_num:1  | left:104  | top:412  | width:389  | height:40   | conf:68 | text:'display_manager':
level:5  | page_num:1  | block_num:3  | par_num:1  | line_num:3  | word_num:2  | left:522  | top:412  | width:442  | height:40   | conf:85 | text:<DisplayManager.X11:
level:5  | page_num:1  | block_num:3  | par_num:1  | line_num:3  | word_num:3  | left:994  | top:415  | width:61   | height:37   | conf:91 | text:2>,
level:5  | page_num:1  | block_num:3  | par_num:1  | line_num:4  | word_num:1  | left:104  | top:462  | width:501  | height:40   | conf:60 | text:'is_briefcase_package':
level:5  | page_num:1  | block_num:3  | par_num:1  | line_num:4  | word_num:2  | left:636  | top:463  | width:128  | height:39   | conf:96 | text:False,
19:49:00 - DEBUG   - normcap.main_window    - L:302 - Apply Magics
19:49:00 - INFO    - normcap.email_magic    - L:33  - 0 emails found ['']
19:49:00 - DEBUG   - normcap.email_magic    - L:44  - 0/410 chars in emails. Ratio: 0.0)
19:49:00 - INFO    - normcap.url_magic      - L:52  - 0 URLs found ['']
19:49:00 - DEBUG   - normcap.url_magic      - L:62  - 0 of 410 chars in emails (ratio: 0.0)
19:49:00 - DEBUG   - normcap.magic          - L:73  - All scores: {'SingleLineMagic': 0, 'MultiLineMagic': 0, 'ParagraphMagic': 80.0, 'EmailMagic': 0.0, 'UrlMagic': 0.0}
19:49:00 - INFO    - normcap.magic          - L:90  - Highest scored magic: ParagraphMagic (80.0)
19:49:00 - DEBUG   - normcap.main_window    - L:304 - Result from applying Magics:
best_magic: ParagraphMagic
image: <PySide2.QtGui.QImage(QSize(2038, 582),format=QImage::Format_RGB32,depth=32,devicePixelRatio=1,bytesPerLine=8152,sizeInBytes=4744464) at 0x7f05b7fbe500>
image_area: 1186116
image_size: (2038, 582)
lines: wee voewws ed ee —s Me ere Lp ee ew ee /plugins' ] 5lSocket: OpenSSL 1.1.1g 21 Apr 2020 True :48:55 - INFO - normeap.utils - L:152 - Copy 0 traineddata files to conf
:48:55 - DEBUG - normcap.app - L:65 - System info:
'config_directory': PosixPath('/home/sojusnik/.config/normcap'),
'desktop_environment': <DesktopEnvironment.GNOME: 1>,
'display_manager': <DisplayManager.X11: 2>,
'is_briefcase_package': False,
mean_conf: 71.96296296296296
mode: 1
num_blocks: 3
num_lines: 4
num_pars: 3
psm_opt: 1
rect: Rect(left=620, top=411, right=1207, bottom=543)
scale_factor: 1.0
scores: {'SingleLineMagic': 0, 'MultiLineMagic': 0, 'ParagraphMagic': 80.0, 'EmailMagic': 0.0, 'UrlMagic': 0.0}
screen: ScreenInfo(is_primary=True, device_pixel_ratio=1.0, geometry=Rect(left=0, top=0, right=1920, bottom=1080), index=0)
text: wee voewws ed ee —s Me ere Lp ee ew ee /plugins' ] 5lSocket: OpenSSL 1.1.1g 21 Apr 2020 True :48:55 - INFO - normeap.utils - L:152 - Copy 0 traineddata files to conf :48:55 - DEBUG - normcap.app - L:65 - System info: 'config_directory': PosixPath('/home/sojusnik/.config/normcap'), 'desktop_environment': <DesktopEnvironment.GNOME: 1>, 'display_manager': <DisplayManager.X11: 2>, 'is_briefcase_package': False,
transformed: wee voewws ed ee —s Me ere Lp ee ew ee

/plugins' ]

5lSocket: OpenSSL 1.1.1g 21 Apr 2020 True

:48:55 - INFO - normeap.utils - L:152 - Copy 0 traineddata files to conf :48:55 - DEBUG - normcap.app - L:65 - System info:

'config_directory': PosixPath('/home/sojusnik/.config/normcap'), 'desktop_environment': <DesktopEnvironment.GNOME: 1>, 'display_manager': <DisplayManager.X11: 2>, 'is_briefcase_package': False,
words: 
level:5  | page_num:1  | block_num:1  | par_num:1  | line_num:1  | word_num:1  | left:108  | top:80   | width:100  | height:7    | conf:0  | text:wee
level:5  | page_num:1  | block_num:1  | par_num:1  | line_num:1  | word_num:2  | left:275  | top:68   | width:113  | height:28   | conf:7  | text:voewws
level:5  | page_num:1  | block_num:1  | par_num:1  | line_num:1  | word_num:3  | left:576  | top:68   | width:62   | height:29   | conf:2  | text:ed
level:5  | page_num:1  | block_num:1  | par_num:1  | line_num:1  | word_num:4  | left:639  | top:80   | width:59   | height:13   | conf:19 | text:ee
level:5  | page_num:1  | block_num:1  | par_num:1  | line_num:1  | word_num:5  | left:1062 | top:80   | width:36   | height:7    | conf:22 | text:—s
level:5  | page_num:1  | block_num:1  | par_num:1  | line_num:1  | word_num:6  | left:1238 | top:80   | width:234  | height:16   | conf:40 | text:Me
level:5  | page_num:1  | block_num:1  | par_num:1  | line_num:1  | word_num:7  | left:1486 | top:80   | width:105  | height:7    | conf:44 | text:ere
level:5  | page_num:1  | block_num:1  | par_num:1  | line_num:1  | word_num:8  | left:1624 | top:80   | width:53   | height:13   | conf:45 | text:Lp
level:5  | page_num:1  | block_num:1  | par_num:1  | line_num:1  | word_num:9  | left:1688 | top:80   | width:101  | height:13   | conf:41 | text:ee
level:5  | page_num:1  | block_num:1  | par_num:1  | line_num:1  | word_num:10 | left:1800 | top:80   | width:43   | height:7    | conf:52 | text:ew
level:5  | page_num:1  | block_num:1  | par_num:1  | line_num:1  | word_num:11 | left:1860 | top:80   | width:49   | height:13   | conf:43 | text:ee
level:5  | page_num:1  | block_num:2  | par_num:1  | line_num:1  | word_num:1  | left:80   | top:105  | width:214  | height:40   | conf:83 | text:/plugins'
level:5  | page_num:1  | block_num:2  | par_num:1  | line_num:1  | word_num:2  | left:264  | top:101  | width:30   | height:58   | conf:54 | text:]
level:5  | page_num:1  | block_num:2  | par_num:2  | line_num:1  | word_num:1  | left:80   | top:155  | width:189  | height:34   | conf:48 | text:5lSocket:
level:5  | page_num:1  | block_num:2  | par_num:2  | line_num:1  | word_num:2  | left:319  | top:159  | width:158  | height:37   | conf:92 | text:OpenSSL
level:5  | page_num:1  | block_num:2  | par_num:2  | line_num:1  | word_num:3  | left:502  | top:159  | width:132  | height:37   | conf:88 | text:1.1.1g
level:5  | page_num:1  | block_num:2  | par_num:2  | line_num:1  | word_num:4  | left:680  | top:159  | width:43   | height:30   | conf:95 | text:21
level:5  | page_num:1  | block_num:2  | par_num:2  | line_num:1  | word_num:5  | left:745  | top:159  | width:67   | height:37   | conf:95 | text:Apr
level:5  | page_num:1  | block_num:2  | par_num:2  | line_num:1  | word_num:6  | left:837  | top:159  | width:88   | height:30   | conf:96 | text:2020
level:5  | page_num:1  | block_num:2  | par_num:2  | line_num:1  | word_num:7  | left:947  | top:159  | width:90   | height:30   | conf:96 | text:True
level:5  | page_num:1  | block_num:2  | par_num:3  | line_num:1  | word_num:1  | left:80   | top:210  | width:128  | height:30   | conf:67 | text::48:55
level:5  | page_num:1  | block_num:2  | par_num:3  | line_num:1  | word_num:2  | left:235  | top:226  | width:15   | height:5    | conf:91 | text:-
level:5  | page_num:1  | block_num:2  | par_num:3  | line_num:1  | word_num:3  | left:278  | top:210  | width:88   | height:31   | conf:96 | text:INFO
level:5  | page_num:1  | block_num:2  | par_num:3  | line_num:1  | word_num:4  | left:459  | top:226  | width:14   | height:5    | conf:92 | text:-
level:5  | page_num:1  | block_num:2  | par_num:3  | line_num:1  | word_num:5  | left:501  | top:207  | width:289  | height:40   | conf:62 | text:normeap.utils
level:5  | page_num:1  | block_num:2  | par_num:3  | line_num:1  | word_num:6  | left:1019 | top:226  | width:14   | height:5    | conf:92 | text:-
level:5  | page_num:1  | block_num:2  | par_num:3  | line_num:1  | word_num:7  | left:1062 | top:210  | width:109  | height:31   | conf:89 | text:L:152
level:5  | page_num:1  | block_num:2  | par_num:3  | line_num:1  | word_num:8  | left:1198 | top:226  | width:15   | height:5    | conf:91 | text:-
level:5  | page_num:1  | block_num:2  | par_num:3  | line_num:1  | word_num:9  | left:1238 | top:210  | width:90   | height:37   | conf:96 | text:Copy
level:5  | page_num:1  | block_num:2  | par_num:3  | line_num:1  | word_num:10 | left:1350 | top:210  | width:23   | height:30   | conf:85 | text:0
level:5  | page_num:1  | block_num:2  | par_num:3  | line_num:1  | word_num:11 | left:1396 | top:207  | width:245  | height:34   | conf:91 | text:traineddata
level:5  | page_num:1  | block_num:2  | par_num:3  | line_num:1  | word_num:12 | left:1665 | top:207  | width:111  | height:34   | conf:96 | text:files
level:5  | page_num:1  | block_num:2  | par_num:3  | line_num:1  | word_num:13 | left:1799 | top:211  | width:45   | height:30   | conf:96 | text:to
level:5  | page_num:1  | block_num:2  | par_num:3  | line_num:1  | word_num:14 | left:1865 | top:207  | width:93   | height:34   | conf:95 | text:conf
level:5  | page_num:1  | block_num:2  | par_num:3  | line_num:2  | word_num:1  | left:80   | top:262  | width:128  | height:30   | conf:67 | text::48:55
level:5  | page_num:1  | block_num:2  | par_num:3  | line_num:2  | word_num:2  | left:235  | top:278  | width:14   | height:4    | conf:92 | text:-
level:5  | page_num:1  | block_num:2  | par_num:3  | line_num:2  | word_num:3  | left:275  | top:262  | width:113  | height:30   | conf:71 | text:DEBUG
level:5  | page_num:1  | block_num:2  | par_num:3  | line_num:2  | word_num:4  | left:459  | top:278  | width:14   | height:4    | conf:87 | text:-
level:5  | page_num:1  | block_num:2  | par_num:3  | line_num:2  | word_num:5  | left:501  | top:268  | width:245  | height:30   | conf:91 | text:normcap.app
level:5  | page_num:1  | block_num:2  | par_num:3  | line_num:2  | word_num:6  | left:1019 | top:278  | width:14   | height:4    | conf:91 | text:-
level:5  | page_num:1  | block_num:2  | par_num:3  | line_num:2  | word_num:7  | left:1062 | top:262  | width:87   | height:30   | conf:43 | text:L:65
level:5  | page_num:1  | block_num:2  | par_num:3  | line_num:2  | word_num:8  | left:1198 | top:278  | width:14   | height:4    | conf:84 | text:-
level:5  | page_num:1  | block_num:2  | par_num:3  | line_num:2  | word_num:9  | left:1240 | top:262  | width:134  | height:36   | conf:96 | text:System
level:5  | page_num:1  | block_num:2  | par_num:3  | line_num:2  | word_num:10 | left:1395 | top:258  | width:106  | height:34   | conf:96 | text:info:
level:5  | page_num:1  | block_num:3  | par_num:1  | line_num:1  | word_num:1  | left:104  | top:309  | width:412  | height:40   | conf:52 | text:'config_directory':
level:5  | page_num:1  | block_num:3  | par_num:1  | line_num:1  | word_num:2  | left:546  | top:309  | width:980  | height:41   | conf:82 | text:PosixPath('/home/sojusnik/.config/normcap'),
level:5  | page_num:1  | block_num:3  | par_num:1  | line_num:2  | word_num:1  | left:104  | top:360  | width:479  | height:41   | conf:51 | text:'desktop_environment':
level:5  | page_num:1  | block_num:3  | par_num:1  | line_num:2  | word_num:2  | left:611  | top:360  | width:577  | height:40   | conf:91 | text:<DesktopEnvironment.GNOME:
level:5  | page_num:1  | block_num:3  | par_num:1  | line_num:2  | word_num:3  | left:1219 | top:364  | width:60   | height:36   | conf:91 | text:1>,
level:5  | page_num:1  | block_num:3  | par_num:1  | line_num:3  | word_num:1  | left:104  | top:412  | width:389  | height:40   | conf:68 | text:'display_manager':
level:5  | page_num:1  | block_num:3  | par_num:1  | line_num:3  | word_num:2  | left:522  | top:412  | width:442  | height:40   | conf:85 | text:<DisplayManager.X11:
level:5  | page_num:1  | block_num:3  | par_num:1  | line_num:3  | word_num:3  | left:994  | top:415  | width:61   | height:37   | conf:91 | text:2>,
level:5  | page_num:1  | block_num:3  | par_num:1  | line_num:4  | word_num:1  | left:104  | top:462  | width:501  | height:40   | conf:60 | text:'is_briefcase_package':
level:5  | page_num:1  | block_num:3  | par_num:1  | line_num:4  | word_num:2  | left:636  | top:463  | width:128  | height:39   | conf:96 | text:False,
19:49:00 - DEBUG   - normcap.clipboard      - L:15  - Copy to clipboard:
wee voewws ed ee —s Me ere Lp ee ew ee

/plugins' ]

5lSocket: OpenSSL 1.1.1g 21 Apr 2020 True

:48:55 - INFO - normeap.utils - L:152 - Copy 0 traineddata files to conf :48:55 - DEBUG - normcap.app - L:65 - System info:

'config_directory': PosixPath('/home/sojusnik/.config/normcap'), 'desktop_environment': <DesktopEnvironment.GNOME: 1>, 'display_manager': <DisplayManager.X11: 2>, 'is_briefcase_package': False,
19:49:00 - ERROR   - normcap.utils          - L:105 - Uncaught exception! Quitting NormCap!
Traceback (most recent call last):
  File "/home/sojusnik/.local/lib/python3.9/site-packages/normcap/gui/main_window.py", line 223, in _notify_or_close
    self.com.on_quit_or_hide("detection completed")
TypeError: native Qt signal is not callable
```

BTW, thanks for creating NormCap! It's a great time saver!

**Comment by dynobo** [comment]
Thanks for the bug report! I'll have to look into that...

**Comment by dynobo** [comment]
Should be fixed in NormCap v0.2.7 :-)
